File: Cpp/linearRegression.cpp
pair<ld, ld> linearRegression(vector<ld> v, vector<ld> u){
    int n = v.size();
    ld sx = 0, sy = 0, sxy = 0, sxx = 0;
    for(int i = 0; i < n; i++) sx += v[i], sy += u[i], sxy += v[i] * u[i], sxx += v[i] * v[i];
    ld m = (n * sxy - sx * sy) / (n * sxx - sx * sx);
    return{m, (sy - sx * m) / n};
}

File: Python/linearRegression.py
def linear_regression(v, u):
            v = np.array(v)
            u = np.array(u)
            n = len(v)
            sx = np.sum(v)
            sy = np.sum(u)
            sxy = np.sum(v * u)
            sxx = np.sum(v * v)
            m = (n * sxy - sx * sy) / (n * sxx - sx * sx)
            return m, (sy - sx * m) / n
